vars/pyenv.groovy

```diff
@@ -4,11 +4,27 @@ import com.ableton.VirtualEnv
 
 /**
  * Create a new {@link com.ableton.VirtualEnv} instance using pyenv.
+ *
+ * This function assumes that the Jenkins instance has a {@code PYENV_ROOT} environment
+ * variable. To manually specify the pyenv root directory, use the
+ * {@link #createVirtualEnv(String, String)} function instead.
+ *
  * @param pythonVersion Python version, as given by pyenv versions --list.
- * @param pyenvRoot Path of the pyenv installation. If {@code null}, then the pyenv root
- *        will be detected from the environment in {@code PYENV_ROOT}.
+ * @param pyenvRoot Path of the pyenv installation.
  * @return New {@link com.ableton.VirtualEnv} instance.
  */
-VirtualEnv createVirtualEnv(String pythonVersion, String pyenvRoot = null) {
-  return new Pyenv(this, pythonVersion, pyenvRoot ?: env.PYENV_ROOT).createVirtualEnv()
```

Comment:
I don't really see the value of 99e07088674a454670c8fbc4ead269cc23116eb9. It only seems to add more code.

Reply:
Ok, we can remove it if you like. See 012ed7a.